subworkflows/nf-core/vcf_gather_bcftools/tests/main.nf.test.snap

```diff
@@ -0,0 +1,124 @@
+{
+    "homo_sapiens - [vcf, tbi] [vcf, tbi] - meta [] 2 - sample - true": {
+        "content": [
+            [
+                "27f188f2e0aeb3af6723aa52beb11edc"
+            ],
+            [
+                "test.vcf.gz.tbi"
+            ],
+            [
+                "versions.yml:md5,185c543cc7cb5a077974b7f30fc4c668",
+                "versions.yml:md5,a1f631d2e625bc6eeba98485d6f36f3b",
+                "versions.yml:md5,e0acb75282e79017ab0cac41793b88b2"
+            ]
+        ],
+        "meta": {
+            "nf-test": "0.9.2",
+            "nextflow": "24.10.4"
+        },
+        "timestamp": "2025-05-20T22:17:10.97599428"
+    },
+    "homo_sapiens no_meta - [vcf, tbi] [vcf, tbi] - meta equal [] 2 - [] - true": {
+        "content": [
+            [
+                "27f188f2e0aeb3af6723aa52beb11edc"
+            ],
+            [
+                "test.vcf.gz.tbi"
+            ],
+            [
+                "versions.yml:md5,185c543cc7cb5a077974b7f30fc4c668",
+                "versions.yml:md5,a1f631d2e625bc6eeba98485d6f36f3b",
+                "versions.yml:md5,e0acb75282e79017ab0cac41793b88b2"
+            ]
+        ],
+        "meta": {
+            "nf-test": "0.9.2",
+            "nextflow": "24.10.4"
+        },
+        "timestamp": "2025-05-20T22:18:35.639623638"
+    },
+    "homo_sapiens no_meta - [vcf, tbi] [vcf, tbi] - meta equal [] 2 - [] - true -- stub": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "groupSize": 2,
+                            "groupTarget": {
+                                "id": "test"
+                            }
+                        },
+                        "test.vcf.gz:md5,68b329da9893e34099c7d8ad5cb9c940"
+                    ]
+                ],
+                "1": [
+                    [
+                        {
+                            "groupSize": 2,
+                            "groupTarget": {
+                                "id": "test"
+                            }
```

Comment:
Aah, same comment as on the somalier subworkflow, we should undo the groupKey.

Reply:
This is fixed !